MAUI/ComboBox/UI-Customization.md

```diff
@@ -661,6 +661,50 @@ The following image illustrates the result of the above code:
 
 ![.NET MAUI ComboBox ItemTemplateSelector](Images/UICustomization/TemplateSelector.png)
 
+## Styling token items
+
+The ComboBox control allows you to customize the style of TokenItem generated in the selection area by using the `TokenItemStyle` property. 
```

Comment:
The ComboBox control allows you to customize the style of TokenItem generated in the selection area using the `TokenItemStyle` property.